components/BUILD.gn

```diff
@@ -8,17 +8,18 @@ import("//build/config/ui.gni")
 import("//testing/test.gni")
 import("//tools/grit/repack.gni")
 
+if (is_ios) {
+  import("//ios/build/config.gni")
+}
+
 test("brave_components_unittests") {
   use_xvfb = use_xvfb_in_this_config
 
-  data = [ "test/data/ai_chat/" ]
-
   deps = [
     "//base",
     "//brave/components/ai_chat/core/browser:unit_tests",
     "//brave/components/ai_chat/core/common:unit_tests",
     "//brave/components/api_request_helper:unit_tests",
-    "//brave/components/brave_wallet/browser:unit_tests",
```

Comment:
I believe this should stay in deps after https://github.com/brave/brave-core/pull/30072

Reply:
That PR is incorrect and should be fixed because `"//brave/components/brave_wallet/browser:unit_tests",` has content deps so I moved it inside `use_blink` below https://github.com/brave/brave-core/pull/29992/files#diff-0213b93d3260f4ad014ca0b858368dc8901b3878d5cdc75f3e0b7ae32c18e570R31

Reply:
sorry, I got something mixed up and was looking at the wrong thing or thought I was looking at the original PR

Reply:
These two fail as there seem no test/data on ios or something related is broken 
`BitcoinSerializer.AddressToScriptPubkey_BitcoinCoreTestVectors`
`HDKeyEd25519Slip23UnitTest.CardanoSdkCryptoSlip23`

This
`CardanoRpcUnitTest.PostTransaction`
hangs due to some issue when lambda in `base::test::RunUntil` is never called. Not sure what's happening there and why it's ios-specific.

I'm ok if these tests (or whole test files if simpler) would be disabled for ios tests by now.

Reply:
adding the relevant files to `data = []` should fix the missing test data on ios